docs/data/migration/migration-data-grid-v7/migration-data-grid-v7.md

```diff
@@ -399,6 +399,10 @@ You have to import it from `@mui/x-license` instead:
 - The icon slots now require material icons to be passed like `Icon as any`.
   Note: This is due to typing issues that might be resolved later.
 
+### Bundling
+
+The Data Grid now requires a bundler that can handle CSS imports.
+
```

Comment:
Let's cover some popular setups here.
- Next.js Pages router: https://github.com/mui/mui-x/issues/17427#issuecomment-2813967605
- Webpack
- Vitest https://github.com/mui/mui-x/issues/17427#issuecomment-2813735154
- ?

Reply:
What level of detail do you have in mind? Just a link?

Reply:
Something like https://mui.com/x/migration/usage-with-material-ui-v5-v6/, with code snippets and links

Reply:
I've added snippets here, but should we move them somewhere in the docs and link to there instead? Maybe the quickstart section?